javascript-practice/src/js/controllers/dashboardController.js

```diff
@@ -24,13 +24,20 @@ export default class DashboardController {
     this.renderTableListUsers();
   }
 
+  /**
+   * Check user access by checking the roles saved in localStorage.
+   * Redirect to the index page if the role is not SUPER_ADMIN.
```

Comment:
For function comment, it should be describe common usecase, a specific comment should be in code content